auth/src/main/java/com/firebase/ui/auth/data/remote/SignInKickstarter.java

```diff
@@ -127,6 +129,16 @@ private void redirectSignIn(String provider, String email) {
                         EmailActivity.createIntent(getApplication(), getArguments(), email),
                         RequestCodes.EMAIL_FLOW)));
                 break;
+            case PhoneAuthProvider.PROVIDER_ID:
+                Bundle args = new Bundle();
+                args.putString(ExtraConstants.PHONE, email);
```

Comment:
Should we rename the `email` arg to `identifier` or something?  Seems strange to see `put(PHONE, email)`